Loaded module: python3/3.6.2
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/util/decorators.py:9: NumbaDeprecationWarning: An import was requested from a module that has moved location.
Import of 'jit' requested from: 'numba.decorators', please update to use 'numba.core.decorators' or pin to Numba version 0.48.0. This alias will not be present in Numba version 0.50.0.
  from numba.decorators import jit as optional_jit
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
/work1/s183921/fagprojekt2020/preprocess/spraakbanken/vae/preprocess_vae/lib/python3.6/site-packages/librosa/core/audio.py:161: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn('PySoundFile failed. Trying audioread instead.')
